marketplaceItems/sitPoint/appResources/json/invisibleSitCube.json

```diff
@@ -0,0 +1,68 @@
+{
+  "DataVersion": 0,
+  "Entities": [
+    {
+    "id": "{11cc3121-4975-457b-8058-6d532013c426}",
+    "type": "Box",
+    "created": "2019-01-16T23:10:19Z",
+    "lastEdited": 1547680198906598,
+    "lastEditedBy": "{77356988-272b-49e6-b445-4768947ca89f}",
+    "dimensions": {
+        "x": 0.20000000298023224,
+        "y": 0.20000000298023224,
+        "z": 0.20000000298023224
+    },
+    "rotation": {
+        "x": -0.0000152587890625,
+        "y": -0.0000152587890625,
+        "z": -0.0000152587890625,
+        "w": 1
+    },
+    "damping": 0,
+    "script": "http://hifi-content.s3-us-west-1.amazonaws.com/Experiences/Releases/marketPlaceItems/sitPoint/2019-01-16_11-07-10/appResources/appData/sitClient.js",
+    "scriptTimestamp": 1547680187707,
+    "serverScripts": "http://hifi-content.s3-us-west-1.amazonaws.com/Experiences/Releases/marketPlaceItems/sitPoint/2019-01-16_11-07-10/appResources/appData/sitServer.js",
```

Comment:
Please replace this link with relative links for GitHub and make a note in the README that you have to change the script for one's own usage.